src/solvers/qbf/qbf_bdd_core.cpp

```diff
@@ -227,10 +240,10 @@ propt::resultt qbf_bdd_coret::prop_solve()
         matrix->nodeCount() << " nodes" << std::endl;
       #endif
 
-      *matrix = matrix->UnivAbstract(*bdd_variable_map[var]);
+      *matrix=matrix->UnivAbstract(*bdd_variable_map[var]);
     }
     else
-      throw ("Unquantified variable");
+      throw("Unquantified variable");
```

Comment:
Will add a rule to the coding standard and add a check to the linter for this unless there is a good reason not to?